datasets/case01/queries/0001.1_fetchObjectById.sql

```diff
@@ -34,5 +34,5 @@ SELECT objectId,iauId,ra_PS,ra_PS_Sigma,decl_PS,decl_PS_Sigma,radecl_PS_Cov,htmI
 	   yDeclOffset_SG,yDeclOffset_SG_Sigma,yRaDeclOffset_SG_Cov,yLnL_PS,yLnL_SG,yFlux_PS,yFlux_PS_Sigma,yFlux_ESG,
 	   yFlux_ESG_Sigma,yFlux_Gaussian,yFlux_Gaussian_Sigma,yTimescale,yEarliestObsTime,yLatestObsTime,ySersicN_SG,
 	   ySersicN_SG_Sigma,yE1_SG,yE1_SG_Sigma,yE2_SG,yE2_SG_Sigma,yRadius_SG,yRadius_SG_Sigma,yFlags
-FROM   Object
+FROM   {DBTAG}.Object
```

Comment:
Putting the database name as a mysql command line argument results in an authentication failure since the proxy is only connected to the local database, which only has the results database. This delays authentication until contacting the central database. 
When running the actual tests, the mysql database name needs to be used to generate the baseline and the qserv database name needs to be used for the test.

Reply:
I'm not sure I can follow this. Authentication should not depend on a database name, it only needs user name and password (and this implies that you have to have authentication info in czar-local server). I can imagine that `USE DATABASE` can fail if you do not have database defined in the local server and when you give default database name to a client then it most likely results in `USE DATABASE`. In any case with multi-czar we still have to support `USE` statement and a default database, I don't think it is right to require from our clients to use database name in every SELECT statement, that would be too restrictive w.r.t. SQL standard that we have to support.
I would be OK with this change if it remained on a separate branch but I can't approve merging this with master. I think that replicating database schema to every czar-local server is the only way to handle it now with the limitation that we have on mysql-proxy side. I think you already have to do something for czar-local server setup (e.g. authentication info and result database creation), you'll just need to add schema setup to that list.

Reply:
@andy-slac I think I understand @jgates108 's motivation. Here is this example:
```
[gapon@lsst-qserv-master01 qserv]$ mysql --protocol=tcp -hlocalhost -P 4040 -uqsmaster mysql
ERROR 1044 (42000): Access denied for user 'qsmaster'@'localhost' to database 'mysql'
```

Reply:
That is because `qsmaster` is not authorized to access `mysql` database. But if `qsmaster` is authorized to use `LSST` dabase then this command must work independently of single-czar or multi-czar setup:

    % mysql --protocol=tcp -hlocalhost -P 4040 -uqsmaster LSST

Reply:
To get that to work in multi-czar will require a lot more work, which does not need to take place in this ticket, and I don't see that particular piece of functionality being essential to qserv. Authentication is still done in the integration tests now, just not by the czar proxy. I made comments about this in the jira ticket.

If it is deemed to be necessary, the proxy can be changed to connect to the central database (or a copy thereof) to get authentication information. Or, we can figure out how to get all the relevant information on all czars and keep it up to date.

Reply:
If you say that latter ticket will undo all these small changes that you have done to 200+ files then  we should not merge this into master at this point and do it later with all other changes. I think we need to talk more about basic design of things, can we re-start this discussion again on Wednesday?

Reply:
Those changes don't need to be undone.